File: Java/src/main/java/com/gildedrose/items/AgedBrieItem.java
package com.gildedrose.items;

import com.gildedrose.Item;

public class AgedBrieItem extends GildedItem {
    public AgedBrieItem(Item item) { super(item); }

    @Override
    public void updateQuality() {
        increaseQuality(1);
        decreaseSellIn();
        if (item.sellIn < 0)
            increaseQuality(1);
    }
}

File: php/src/Items/AgedBrieItem.php
<?php

declare(strict_types=1);

namespace GildedRose\Items;

use GildedRose\Item;
use GildedRose\Items\Abstract\AbstractItem;
use GildedRose\Items\Interface\ItemInterface;

class AgedBrieItem extends AbstractItem implements ItemInterface
{
    /**
     * @param Item $item
     *
     * @return void
     */
    public function updateQuality(Item $item): void
    {
        // "Aged Brie" actually increases in Quality the older it gets
        $this->increaseQuality($item, $this->getQualityDecrease($item));
    }
}
